make/modules/java.desktop/lib/Awt2dLibraries.gmk

```diff
@@ -144,25 +139,26 @@ $(eval $(call SetupJdkLibrary, BUILD_LIBAWT, \
     DISABLED_WARNINGS_microsoft := 4244 4996, \
     DISABLED_WARNINGS_microsoft_awt_Toolkit.cpp := 4267, \
     LDFLAGS_macosx := -L$(INSTALL_LIBRARIES_HERE), \
-    LDFLAGS_windows := -delayload:user32.dll -delayload:gdi32.dll \
-        -delayload:shell32.dll -delayload:winmm.dll \
-        -delayload:winspool.drv -delayload:imm32.dll \
-        -delayload:ole32.dll -delayload:comdlg32.dll \
-        -delayload:comctl32.dll -delayload:shlwapi.dll, \
-    LIBS_unix := -ljvm -ljava $(LIBM), \
-    LIBS_linux :=  $(LIBDL), \
+    LDFLAGS_windows := -delayload:comctl32.dll -delayload:comdlg32.dll \
+        -delayload:gdi32.dll -delayload:imm32.dll -delayload:ole32.dll \
+        -delayload:shell32.dll -delayload:shlwapi.dll -delayload:user32.dll \
+        -delayload:winmm.dll -delayload:winspool.drv, \
```

Comment:
I suppose (presume?) that delayload isn't sensitive to the order ?
But I do have to ask if you ran all the client automated tests, as well as making sure builds work ?
And surely logical ordering related to dependencies is more important than lexical sort order ?

Reply:
`/delayload` is not order sensitive, no.

I'm not sure what you mean by "logical ordering related to dependencies". All these are dependencies for awt.dll, and there is no intra-dependency relation between them.

What you see here is just a single file -- I have gone through the entire build system to make sure all calls to `SetupJdkLibrary` has been consistent. Among the things I have standardized is making sure that all libraries are listed in alphabetical order. It will not affect the final product but will help us keep the makefile source code clean and consistent.

Reply:
I have run tier 1 + tier 3 on all platforms supported on the Oracle CI system, in addition to the comparison build that shows that the resulting binaries should have only superficial changes. 

Please advice if you think I need to run any additional testing.